hsclient/hydroshare.py

```diff
@@ -905,9 +907,14 @@ def resource(self, resource_id: str, validate: bool = True) -> Resource:
         :param validate: Defaults to True, set to False to not validate the resource exists
```

Comment:
Github doesn't allow me to create a suggestion further above, but the use_cache parameter would also have to be added above.  Then the use_cache method would be checked before reading from and writing to the cache.

```suggestion
        :param validate: Defaults to True, set to False to not validate the resource exists
        :param use_cache: Defaults to True, set to False to skip the cache, and always retrieve the resource from HydroShare.  This parameter also does not cache the retrieved Resource object.
```

Reply:
A test would have to be added to check this parameter as well.